### DISCUSSION TITLE
We are on ProductHunt!

### DISCUSSION THREAD

**Discussion by Ibrahim-Halil-Kuray** [ProxeusApp/community#34] [General]
- [Reviews](https://www.producthunt.com/products/proxeus/reviews) on ProductHunt.
- [Comments and Questions](https://www.producthunt.com/products/proxeus#proxeus)  about Proxeus

**Comment by Ibrahim-Halil-Kuray** [comment]
[@tomas_blatak1](https://www.producthunt.com/@tomas_blatak1)

Can I use your platform in a similar way as ciphers with public and secret keys? Or what is a typical use case for "non-blockchain" guy ?

**Reply by loleg** [reply]
Proxeus is not a drop-in replacement for encryption tools like PGP or nacl/box, though it could be a good complement for improving information security in your team. The typical use case is to collect data from customers or partners, and generate reports which can be shared or exported in PDF or Office formats. You only need the blockchain tools in order to sign (or counter-sign) the PDF editions.

**Comment by Ibrahim-Halil-Kuray** [comment]
[@fabian_untermoser](https://www.producthunt.com/@fabian_untermoser)

Cool idea. Reminds me of a similar product: Sproof

If you know it, can you tell me how your product differs?

I think Sproof transitioned away from blockchain now, so that would still be your USP when it comes to document signing.

**Reply by loleg** [reply]
Thanks! We think so too. Although we have not used it, sproof sign looks like a commercial service based in Austria for adding signatures to documents in a way that complies with European laws. We would be happy to take it for a spin!

Proxeus is an open source alternative with fewer features out of the box, that allows you to deploy an e-signature solution on-premises or with an IT partner anywhere in the world - and customize it to fit your own legal requirements.

**Comment by Ibrahim-Halil-Kuray** [comment]
[@andreasmurk](https://www.producthunt.com/@andreasmurk)

Which exact blockchain applications can be created using your product?

**Reply by loleg** [reply]
A number of 'dapps' and prototypes have been developed on top of the Proxeus framework. It makes it easy for business users to chain together simple components (like an e-mail sender, data collection from an external API, or a KYC service) with a point-and-click interface. Proxeus was designed in Switzerland with a lot of attention to usability, accessibility, and internationalization, which puts it in the class of (usually quite expensive) enterprise software products.

The issue of form-based data collection and report generation is a basic need of many business applications, so Proxeus can be a great choice for organizations that would like to add the security and transparency of the blockchain to their digital strategy, without the overhead of developing all these application features from scratch.
